protos/dcs/coalition/v0/coalition.proto

```diff
@@ -77,25 +77,26 @@ message AddGroupRequest {
       ALTITUDE_TYPE_RADIO = 2;
     }
 
-    enum Type {
-      TYPE_UNSPECIFIED = 0;
-      TAKEOFF = 1;
-      TAKEOFF_PARKING = 2;
-      TURNING_POINT = 3;
-      TAKEOFF_PARKING_HOT = 4;
-      LAND = 5;
+    enum PointType {
+      POINT_TYPE_UNSPECIFIED = 0;
+      POINT_TYPE_TAKEOFF = 1;
+      POINT_TYPE_TAKEOFF_PARKING = 2;
+      POINT_TYPE_TURNING_POINT = 3;
+      POINT_TYPE_TAKEOFF_PARKING_HOT = 4;
+      POINT_TYPE_LAND = 5;
     }
 
     dcs.common.v0.Position position = 1;
     AltitudeType altitude_type = 2;
-    Type type = 3;
+    PointType type = 3;
     string action = 4;
     string form = 5;
     double speed = 6;
   }
 
   enum Skill {
-    SKILL_RANDOM = 0;
+    // unspecified skill would use DCS random skill
+    SKILL_UNSPECIFIED = 0;
```

Comment:
I think this is a practical middle ground -- if unspecified, you get the default which is random in DCS MSE.

Reply:
I don't like the "Unspecified means something unexpected". We should increment (and throw error on 0) or leave 0 as RANDOM and have the linter ignore using a comment.

Reply:
I've created an exclusion on the rule for this.